Intersect (Core)/GameObjects/NpcBase.cs

```diff
@@ -30,6 +29,24 @@ public class NpcBase : DatabaseObject<NpcBase>, IFolderable
 
         [NotMapped] public int[] VitalRegen = new int[(int) Vitals.VitalCount];
 
+        [NotMapped]
+        public List<StatusTypes> Immunities = new List<StatusTypes>();
+
+        [JsonIgnore]
+        [Column("Immunities")]
+        public string ImmunitiesJson
+        {
+            get => JsonConvert.SerializeObject(Immunities);
+            set
+            {
+                Immunities = JsonConvert.DeserializeObject<List<StatusTypes>>(value ?? "");
+                if (Immunities == null)
+                {
+                    Immunities = new List<StatusTypes>();
```

Comment:
`?? new List` at the end of the Deserialize line, no reason to have an if statement for this

Reply:
Done